src/Tribe/Customizer/Global_Elements.php

```diff
@@ -129,7 +129,10 @@ public function setup() {
 		$description         = $views_v2_is_enabled ? '' : esc_html__( 'Options selected here will override what was selected in the "General Theme" section.', 'the-events-calendar' );
 
 		$this->defaults = [
-			'link_color' => '#141827',
+			'link_color'               => '#141827',
+			'background_color'         => '#e9e4f0', // TODO: use customizer setting
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 6 space(s) between "'background_color'" and double arrow, but found 9.
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: f8795d74da2b91ebb6b47a9ef562d5c2</sub>